src/calendar.ts

```diff
@@ -128,7 +128,7 @@ export enum Step {
                             {{event.endTime|date: 'HH:mm'}}
                         </span>
                     <span *ngIf="event.allDay" class="monthview-eventdetail-timecolumn">{{allDayLabel !== undefined ? allDayLabel : 'All day'}}</span>
-                    <span class="event-detail"> <span *ngIf="allDayLabel === undefined || allDayLabel.length">|</span>  {{event.title}}</span>
+                    <span class="event-detail"> <span *ngIf="allDayLabel === undefined || allDayLabel && allDayLabel.trim().length">|</span>  {{event.title}}</span>
```

Comment:
I don't think it's necessary to add this logic. It's the responsibility of the users to provide a reasonable value.